apps/nextra/pages/en/build/sdks/ts-sdk/fetch-data-via-sdk.mdx

```diff
@@ -22,13 +22,17 @@ Many queries have a parameter named `options` to customize the results, use it t
 The `Aptos` client can out of the box query both network data from [fullnodes](https://api.mainnet.aptoslabs.com/v1/spec#/) and the [Indexer](https://cloud.hasura.io/public/graphiql?endpoint=https://api.mainnet.aptoslabs.com/v1/graphql) API which contains aggregated and enriched data. If you want to use a custom query for Indexer API data, you can use `aptos.queryIndexer` like so:
 
 ```ts filename="fetch-data.ts"
-const ledgerInfo = aptos.queryIndexer({
- query: `query MyQuery {
-     ledger_infos {
-       chain_id
-     }
-   }`;
-})
+  const ledgerInfo = await aptos.queryIndexer({
+    query: {
+      query: `
+        query MyQuery {
+          ledger_infos {
+            chain_id
+          }
+        }
+      `
```

Comment:
Is this intentional? I'm not sure why there's nested `query` here. Cc @gregnazario who reviewed before

Reply:
Yes, this is exactly what the fix is, which is why I approved this.